website/docs/r/s3_bucket_policy.html.markdown

```diff
@@ -60,5 +60,5 @@ No additional attributes are exported.
 S3 bucket policies can be imported using the bucket name, e.g.,
 
 ```
-$ terraform import aws_s3_bucket_policy.example my-bucket-name
+$ terraform import aws_s3_bucket_policy.allow_access_from_another_account my-bucket-name
```

Comment:
Ooo and this can match the bucket name used in the example too if you want to add it to this PR directly 😄 
```suggestion
$ terraform import aws_s3_bucket_policy.allow_access_from_another_account my-tf-test-bucket
```